apps/app/src/components/LoginForm/LoginForm.tsx

```diff
@@ -172,9 +172,15 @@ export const LoginForm = (props: LoginFormProps): JSX.Element => {
     const loginErrorElementWithDangerouslySetInnerHTML = generateDangerouslySetErrors(loginErrorListForDangerouslySetInnerHTML);
     // Generate login error elements using <ul>, <li>
 
-    const loginErrorElement = props.externalAccountLoginError != null
-      ? generateSafelySetErrors([...loginErrorList, props.externalAccountLoginError])
-      : generateSafelySetErrors(loginErrorList);
+    let loginErrorElement;
+    if (loginErrorList == null || loginErrorList.length === 0) {
+      if (props.externalAccountLoginError != null) {
+        loginErrorElement = generateSafelySetErrors([props.externalAccountLoginError]);
+      }
+    }
+    else {
+      loginErrorElement = generateSafelySetErrors(loginErrorList);
+    }
```

Comment:
こんな感じで

```
    const loginErrorElement = (loginErrorList ?? []).length > 0
      // prioritize loginErrorList because the list should contains new error
      ? generateSafelySetErrors(loginErrorList)
      : generateSafelySetErrors(props.externalAccountLoginError != null ? [props.externalAccountLoginError] : []);
```